static/js/controllers/inbox.js

```diff
@@ -25,9 +25,10 @@ var feedback = require('../modules/feedback'),
       $window,
       APP_CONFIG,
       Auth,
+      ContactSchema,
+      CONTACT_TYPES,
```

Comment:
Well… I mean. Maybe not by your standards! (Don't worry, there is a sublime text shortcut for this!)